testapp/src/main/java/com/rakuten/tech/mobile/testapp/analytics/rat_wrapper/IRATFragment.kt

```diff
@@ -0,0 +1,32 @@
+package com.rakuten.tech.mobile.testapp.analytics.rat_wrapper
+
+interface IRATFragment {
+    /**
+     * Prepare the component's rat event before it is sent using eventTypes, actions,
+     * screen name, miniapp info
+     */
+    fun prepareScreenAppearEventToSend()
```

Comment:
To me, it could be renamed to `setDefaultRatEvent` & also, we can exclude this function from `IRATFragment` and make it as private inside `RATFragment` - same concept in `IRATActivity`, so we don't have to `interface` something default.